docs/content/v2025.1/explore/transactions/_index.md

```diff
@@ -21,10 +21,10 @@ The following table summarizes the support for transactions across the YSQL and
 | Distributed transactions | Yes | Yes | Perform multi-row or multi-table transactions.<br/>An application can connect to any node of the cluster. |
 | Isolation levels | Serializable<br/>Snapshot<br/>Read Committed | Snapshot | Repeatable read isolation level in PostgreSQL maps to snapshot isolation in YSQL. |
 | `AUTOCOMMIT = false` setting | Yes | No | The transaction must be expressed as one statement in YCQL. |
+| Explicit locking | Yes | No | Ability to perform row- and table-level locking |
+| {{<tags/feature/tp>}} Transactional DDL | Transaction per DDL statement  | No | Each DDL statement is a transaction in YSQL, even if other DDL statements are in a transaction block. |
```

Comment:
Shouldn't second column be 'Yes' with description explaining whatever that means